code/modules/reagents/chemistry_machinery/acid_harness.dm

```diff
@@ -42,7 +42,7 @@
 
 /obj/item/clothing/accessory/storage/black_vest/acid_harness
 	name = "A.C.I.D. Harness"
-	desc = "Automated Chemical Integrated Delivery Harness, or really just a franken webbing made by a researcher with poor tailoring skills."
+	desc = "Automated Chemical Integrated Delivery Harness, or really just a franken webbing made by a researcher with poor tailoring skills. Can be configured with a multitool."
```

Comment:
Aren't multitools now called 'Security Access Tuner'?

Reply:
yes... but the naming is super inconsistent... (multitool is scattered everywhere in the codebase)

I should probably do another PR later to fix them all